library/src/test/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapterTest.java

```diff
@@ -590,6 +590,33 @@ public void getFooterPositionInAdapterUsingTag_withAdapterWithManySections_retur
         assertThat(result, is(21));
     }
 
+    @Test
+    public void indexOf() {
+        StatelessSectionStub statelessSectionStub_0 = new StatelessSectionStub(ITEMS_QTY);
+        StatelessSectionStub statelessSectionStub_1 = new StatelessSectionStub(ITEMS_QTY);
+        StatelessSectionStub statelessSectionStub_2 = new StatelessSectionStub(ITEMS_QTY);
+        StatelessSectionStub statelessSectionStub_3 = new StatelessSectionStub(ITEMS_QTY);
```

Comment:
checkstyle is [failing](https://travis-ci.org/luizgrp/SectionedRecyclerViewAdapter/builds/431023776?utm_source=github_status&utm_medium=notification) because of these variable names